src/index.html

```diff
@@ -11,7 +11,183 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="div-with-input">
+          <label for="surname">Surname:</label>
+          <input
+            id="surname"
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="div-with-input">
+          <label for="name">Name:</label>
+          <input
+            id="name"
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="div-with-input">
+          <label for="age">How old are You?</label>
+          <input
+            id="age"
+            class="age"
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+
+        <div class="div-with-input">
+          <label for="birth">Full date of birth:</label>
+          <input
+            id="birth"
+            type="date"
+            name="birth"
+          >
+        </div>
+
+        <div>
+          <label for="accept">I accept the term of the agreement</label>
+          <input
+            id="accept"
+            type="checkbox"
+            name="accept"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="div-with-input">
+          <label for="email">E-mail:</label>
+          <input
+            id="email"
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            autocomplete="off"
+          >
+        </div>
+
+        <div>
+          <label for="password">Password:</label>
+          <input
+            id="password"
+            type="password"
+            name="password"
+            required
+            minlength="5"
+            maxlength="15"
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="div-with-input">
+          <label for="yes">Do you love cats?</label>
+          <input
+            id="yes"
+            type="radio"
+            name="love-cats"
```

Comment:
Check this point:
_Use camelCase for values of name attribute - they should be valid as JavaScript object keys. It should not contain spaces, “-” or other special characters._

- [ ] Change name attr

Reply:
I renamed this name to `loveCats`.